PlayerControls/sources/DefaultControlsViewController.swift

```diff
@@ -569,8 +613,9 @@ public final class DefaultControlsViewController: ContentControlsViewController
                 seekBackButton.isHidden = false
                 seekBackButton.alpha = 1
             default:
-                guard seekForwardButton.layer.animationKeys() == nil else { return }
+                guard seekBackButton.layer.animationKeys() == nil else { return }
```

Comment:
It was `seekForwardButton` in previous implementation - now it is `seekBackButton` - is it ok?

Reply:
Sure, it was a mistake. Cause we're working in renderSeekBack func, and somehow we got seekForward there.